lib/smart_app_launch/smart_access_brands_validate_endpoints_test.rb

```diff
@@ -10,6 +10,9 @@ class SMARTAccessBrandsValidateEndpoints < Inferno::Test
       by checking if it is referenced by at least 1 Organization resource.
       )
 
+    verifies_requirements 'hl7.fhir.uv.smart-app-launch_2.2.0@409',
+                          'hl7.fhir.uv.smart-app-launch_2.2.0@417'
```

Comment:
I would remove both of these - for @409, doesn't verify anything to do with identifiers. for @419, it skips if nothing present, but this is really tested elsewhere.